### ISSUE TITLE
Passlib exception on quickstart.py

### ISSUE THREAD

**Issue by Tigge** [jensl/critic#104]
When running `quickstart.py` from master on my machine machine, I get the following error:

```
tigge@bacchus:/code/critic (master)$ python ./quickstart.py -s /code/critic-state/
Compiling all sources ... done.
Initializing database ... done.
Installing the system ... done.
Shutting down ...
Traceback (most recent call last):
  File "./quickstart.py", line 279, in <module>
    password=auth.hashPassword(arguments.admin_password))
  File "/code/critic/src/auth/__init__.py", line 82, in hashPassword
    return createCryptContext().encrypt(password)
  File "/code/critic/src/auth/__init__.py", line 59, in createCryptContext
    **kwargs)
  File "/usr/lib/python2.7/dist-packages/passlib/context.py", line 1707, in __init__
    self.load(kwds)
  File "/usr/lib/python2.7/dist-packages/passlib/context.py", line 1896, in load
    config = _CryptConfig(source)
  File "/usr/lib/python2.7/dist-packages/passlib/context.py", line 1020, in __init__
    self._init_default_schemes()
  File "/usr/lib/python2.7/dist-packages/passlib/context.py", line 1267, in _init_default_schemes
    raise ValueError("default scheme cannot be deprecated")
ValueError: default scheme cannot be deprecated
```

**Comment by jensl** [comment]
Yeah, I noticed this myself some days ago. Newer versions of passlib (the one in Ubuntu 15.10, for instance) complain about Critic using it wrong, which it did because of a bug in the quickstart installation mechanism.

Fix uploaded here: https://critic-review.org/r/357

**Comment by Tigge** [comment]
Fix is commited as 12e5ffa1a793c0d225ed9bde1963718464d73ae0.
